docker/web-agent/Dockerfile

```diff
@@ -0,0 +1,37 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Start from Java 8 based on Alpine Linux image (~5Mb)
+FROM openjdk:8-jre-alpine
+
+# Settings
+USER root
+ENV AGENT_HOME /opt/ignite/ignite-web-agent
+WORKDIR /opt/ignite
+
+# Add missing software
+RUN apk --update --no-cache add bash
+
+# Copy main binary archive
+COPY ignite-web-agent* ignite-web-agent
+
+# Update permissions
+RUN chmod -R 777 ${AGENT_HOME}
```

Comment:
Why you apply such wide permission for all files in agent folder. May be you want `RUN chmod +x ${AGENT_HOME}/ignite-web-agent.sh`

Reply:
I think it makes sense to follow @nva suggestion if agent doesn't add/copy/move/ files from ${AGENT_HOME}.

Reply:
I will experiment with image built with already `chmod +x ignite-web-agent.sh`